src/picosvg/svg_types.py

```diff
@@ -258,6 +258,7 @@ def callback(curr_xy, cmd, args, prev_xy, prev_cmd, prev_args)
           returns sequence of (new_cmd, new_args) that replace cmd, args
         """
         curr_pos = Point()
+        return_pos = curr_pos  # where a z will take you
```

Comment:
you don't like `move_pos`? How about `start_pos` then? It's the start of the current sub-path.

Reply:
+1 for start_pos. In fact I might go all the way to subpath_start_pos!